### PR TITLE
Feat: Added light theme to my CL

### PR DESCRIPTION
**Pull request by zoef-yt**
Added light theme to my CL. 
I started making my CL with dark mode now change it to light mode.
# Dark mode
image
# Light mode
image

### CODE REVIEW

File: pages/getting-started/typography/typography.html [hunk 1]
```diff
@@ -41,5 +41,6 @@ <h6>h6. UI Game Heading</h6>
 		</div>
 
 		<script src="/prism/prism.js"></script>
+		<script src="/app.js"></script>
```
Comment (sanjitsarkar) [inline pages/getting-started/typography/typography.html:44]: You can script inside head tag by using defer keyword which will only load after the DOM is completely rendered

**Review by sanjitsarkar — APPROVED** [review]
Everything looks good

**Review by Muzammil-khan710 — APPROVED** [review]
Effects for light theme are really awesome.
Good to merge !

**Review by archit7448 — APPROVED** [review]
Good work , looks good to merge

**Review by Megha-Joshi — APPROVED** [review]
Looking awesome! Good to merge.

File: pages/getting-started/typography/typography.html (hunk 1 above)
Comment (zoef-yt) [reply pages/getting-started/typography/typography.html:44]: okay, I will do that from the next project.

**Pull request merged**